src/nvim/cursor_shape.c

```diff
@@ -64,6 +64,9 @@ Array mode_style_array(void)
       PUT(dic, "blinkoff", INTEGER_OBJ(cur->blinkoff));
       PUT(dic, "hl_id", INTEGER_OBJ(cur->id));
       PUT(dic, "id_lm", INTEGER_OBJ(cur->id_lm));
+      PUT(dic, "attr_id", INTEGER_OBJ(cur->id ? syn_id2attr(cur->id) : 0));
+      PUT(dic, "attr_id_lm", INTEGER_OBJ(cur->id_lm ? syn_id2attr(cur->id_lm)
+                                         : 0));
```

Comment:
usually aligned with the `?`

Reply:
the linter complained, but guess I could ignore it.